src/main/webapp/scripts/controller.js

```diff
@@ -0,0 +1,47 @@
+import { makeApiCall } from '/scripts/comments.js';
+import { authorizeButton, signoutButton, commentButton, videoId, commentElement } from '/scripts/view.js';
+import { successfulApiCallback, failedApiCallback, updateSigninStatus } from '/scripts/view.js';
```

Comment:
how did you get this part working? I'm unable to call functions in modules, can you teach me :grinning:

Reply:
sure!